lint-staged.config.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
-  '*.{js,jsx,ts,tsx}': ['eslint --fix', 'eslint'],
+  '*.{js,jsx,ts,tsx}': [
+    'eslint --cache .eslintcache . --fix',
+    'eslint --cache .eslintcache .',
```

Comment:
In the project, it usually relies on the default value and the cache folder is already `.eslintcache`. I don't think it's necessary to explicitly indicate `.eslintcache`. Instead just use the flag `--cache` is enough.

Another question, it's very strange to add the `.` (dot) in lint-staged. The idea of lint-staged is to run lint on files that will be committed and not on the whole project. By adding the `.`, it'll break the benefit of lint-staged.